pymap/backend/maildir/__init__.py

```diff
@@ -67,6 +67,8 @@ def add_subparser(cls, name: str, subparsers: Any) -> ArgumentParser:
                             help='maximum number of IO workers')
         parser.add_argument('--layout', metavar='TYPE', default='++',
                             help='maildir directory layout')
+        parser.add_argument('--colon', metavar='CHAR', default=':',
```

Comment:
Can you change this to `default=None` here? And then of course change the typing from `str` to `Optional[str]` and add an `if ... is not None` check before assigning to `maildir.colon` below. It doesn't make any functional difference, but it just keeps the scope of regression a bit smaller.